packages/owlbot-bootstrapper/common-container/test/split-repo.test.ts

```diff
@@ -149,9 +155,14 @@ describe('SplitRepo class', async () => {
   });
 
   it('should create an empty PR', async () => {
-    const scope = nock('https://api.github.com')
-      .post('/repos/googleapis/fakeRepo/pulls')
-      .reply(201);
+    const scopes = [
+      nock('https://api.github.com')
+        .get('/repos/googleapis/googleapis-gen/commits')
+        .reply(201, {sha: '6dcb09b5b57875f334f61aebed695e2e4193db5e'}),
+      nock('https://api.github.com')
+        .post('/repos/googleapis/fakeRepo/pulls')
+        .reply(201),
```

Comment:
you can chain these together if they use the same host, rather than creating an array yourself.